tests/conftest.py

```diff
@@ -25,13 +25,33 @@ class CustomResourceConfig(ResourceConfig):
     list_route = "/custom/"
 
 
+class CustomResourceRequestContext(ResourceRequestCtx):
+    """Custom resouce request context."""
+
+    def inject_original_request(self):
+        """."""
+        super(CustomResourceRequestContext, self).inject_original_request()
+        self.original_request.update({
+            'content_type': "application/json"
+        })
+
+
+class CustomResourceWithContextConfig(CustomResourceConfig):
+    """Custom resource configuration with custom context."""
+
+    item_route = "/only-json-content/<id>"
+    list_route = "/only-json-content/"
+    format_map = dict(json="application/json")
```

Comment:
Just FYI, `dict` is proven to be up to 6 times slower than `{}`, even if it's a small thing I would default to use `{}`